about/index.html

```diff
@@ -108,7 +108,7 @@ <h1>About</h1>
           <br>
           <p> 
             Databased is headed by Masavir Khaliq, an M.Tech. student at CSA, IISc, and is run by students
-            of the B.Tech. (Maths and Computing) program. We are fortunate to have Chiranjib Bhattacharya (Chair, CSA) as our supervisor.
+            of the B.Tech. (Maths and Computing) program. We are fortunate to have Chiranjib Bhattacharya (Former Chair, CSA) as our supervisor.
```

Comment:
Ah yes! Let us hope we also get in touch with Prof. Vinod :)

Reply:
We certainly will in UMC102!